TDDPracticeTests/TDDPracticeTests/CalculateNumber/CalculatorTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace TestCasesProject.CalculatorCaseStudy
+{
+
+    public class CalculatorTests : IDisposable
+    {
+        private readonly ITestOutputHelper _output;
+        public CalculatorTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+
+        //dispose all running instances
+        public void Dispose()
```

Comment:
What is the purpose of IDisposable inheritance?

Reply:
I used to dispose the instance of test class, I have seen this in an microsoft official video. Sometime its consider in best practices.